framework/src/modules/chain/synchronizer/fast_chain_switching_mechanism.js

```diff
@@ -317,6 +317,7 @@ class FastChainSwitchingMechanism extends BaseSynchronizer {
 				const blockInstance = await this.processor.deserialize(block);
 				await this.processor.processValidated(blockInstance);
 			}
+			this.logger.info('Successfully switched chains. Node is now up to date');
```

Comment:
It would be nice to know the details of the node up to date. something like `this.logger.info({ highestCommonBlockHeight, lastHighestBlockHeight }, 'Successfully switched chains. Node is now up to date');`

Reply:
Done :)